### DISCUSSION TITLE
Beginner's problem, WPF UI NuGet installation and XAML configuration in App.xaml

### DISCUSSION THREAD

**Discussion by zippedalo** [lepoco/wpfui#556] [Q&A] [answered]
i am a new and i have trouble in the 1st step, which asks me to add the directory to app.xaml.i add the nuget pack and the extension but it shows unavailble sign under the code.some tips says the type application is not supported to redict link.what can i do？

**Comment by zippedalo** [comment]
i know little about wpf. so if u can i hope u can give me a complete steps to prepare for it and i really thank you a lot.

**Comment by pomianowski** [comment, marked as answer]
Hey @zippedalo, it would be easier if you could include some code or a sample repository so we can see what's wrong together more easily.
Some basic stuff is described on **Getting started** page https://wpfui.lepo.co/tutorial/getting-started.html

Before you follow the steps in **Getting started** you need to properly add NuGet to your WPF project, more about it here:
https://wpfui.lepo.co/tutorial/nuget.html

It takes some time to understand the basics, I recommend reading the following:
 - [An introduction to NuGet](https://learn.microsoft.com/en-us/nuget/what-is-nuget)
 - [Package consumption workflow](https://learn.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow#ways-to-install-a-nuget-package)
 - [XAML overview](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/xaml-overview)
 - [Create a simple WPF application with C#](https://learn.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-wpf?view=vs-2022)
 - [ResourceDictionary and XAML resource references](https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-resource-dictionary)

If you still have problems, send a snippet of your code and we'll see what could be wrong.

**Reply by zippedalo** [reply]
![image](https://user-images.githubusercontent.com/59652717/225906783-271f0e82-86fe-4345-a392-1ceead858dbe.png)
a screenshot of installing the Nuget package. The main idea of the error output is the .NET SDK i am using (6.0)is not supported to be a target, and it asked me to lower my version of the .NET SDK (5.0). I see others can use the .NET  6, so i wonder whats the problem.

**Reply by zippedalo** [reply]
added: i am using a WPFUI template, not a WPF standard template. Is there anything wrong?

**Reply by pomianowski** [reply]
Hey @zippedalo, the screenshot doesn't tell me much. If you want us to be able to collectively make sense of what might be wrong, create a repository with sample code in which you can reproduce this error, and then include a link to that repository.

**Reply by zippedalo** [reply]
oh iam so thankful to your patience. i have found the problem. thx for your teaching.
